### DISCUSSION TITLE
Mermaid editor improvements

### DISCUSSION THREAD

**Discussion by quank123wip** [Milkdown/milkdown#390] [Ideas]
Mermaid.js has published a new live editor for mermaid. It seems a good way to improve user experience. But unfortunately it's not stable yet, and it maybe not a major goal of this project. I wonder if we can make this a future plan after we have enough time and resources to do this. If it comes to the stable stage, I would like to do this, improving my coding skills and understanding of this project.

Also,  Forgive that I wouldn't open a issue to track its progress until I have time. If any contributor would like to take over this, I'm happy to see this. Just open an issue and mention me.

**Comment by Saul-Mirone** [comment]
Let's wait for the stable release of it first. Then we can start some work.
